atest/testdata/test_libraries/logging_with_logging.robot

```diff
@@ -37,3 +37,6 @@ Timestamps are accurate
 Logging when timeout is in use
     [Timeout]    5 seconds
     Log something
+
+Log with format
+    Log with format
```

Comment:
Newline missing at the end of the file. Good idea to configure your IDE to add them automatically. Not a biggie but will create unnecessary diff later if/when this file is modified.